cherrypy/process/servers.py

```diff
@@ -121,6 +121,14 @@ def index(self):
 import sys
 import time
 import warnings
+import contextlib
+
+import portend
+
+
+class Timeouts:
```

Comment:
Why not? Zen of python says "Namespaces are one honking great idea -- let's do more of those!"

What would be better? A namedtuple?

```
timeouts = collections.namedtuple('timeouts', 'occupied free')(5, 1)
```

Nah, that's impossible to read.

In any other language (C, C++, Java), you would group constants in an enum or struct or class. It seems perfectly sane to do the same in Python, applying DRY to the scope which is "constants about timeouts". It's codacy that's at fault here, demanding docstrings on classes which are simple and self-documenting.